docs/manifest.json

```diff
@@ -1664,7 +1664,24 @@
                     ]
                   },
                   {
-                    "title": "App Router References",
+                    "title": "Nextjs 15+ App Router References",
+                    "items": [
+                      [
+                        {
+                          "title": "Nextjs 15 `auth()`",
+                          "wrap": false,
+                          "href": "/docs/references/nextjs/nextjs15-auth"
+                        },
+                        {
+                          "title": "Nextjs 15 `<ClerkProvider>`",
+                          "wrap": false,
+                          "href": "/docs/references/nextjs/nextjs15-clerkprovider"
+                        }
+                      ]
+                    ]
+                  },
+                  {
+                    "title": "Nextjs <15 App Router References",
```

Comment:
Instead of segmenting the pages way, let's adjust the content within each page to mention the new functionality. We generally want to have our content reflect the latest stable release, with mentions of behavior in prior versions as necessary.